docs/Transactions.md

```diff
@@ -17,14 +17,18 @@ You initialize a transaction by making an async call to `producer.transaction()`
 > - The producer must wait for acknowledgement from all replicas (acks=-1)
 > - The producer must have unlimited retries
 
-Configure the producer client with `maxInFlightRequests: 1` and `idempotent: true` to guarantee EOS. Configuring the two options will enable the settings mentioned above.
+Configure the producer client with `maxInFlightRequests: 1`, `idempotent: true` and a `transactionalId: 'name'` to guarantee EOS. Configuring the options will enable the settings mentioned above.
```

Comment:
Make sure that the value here corresponds to the one down in the example code, so that it's easy for people to make the connection. `transactional-producer` is probably a better example value than `name`.